src/ring_theory/power_basis.lean

```diff
@@ -338,33 +338,71 @@ noncomputable def alg_hom.fintype (pb : power_basis A S) :
 by letI := classical.dec_eq B; exact
 fintype.of_equiv _ pb.lift_equiv'.symm
 
-/-- `pb.equiv pb' h` is an equivalence of algebras with the same power basis. -/
-noncomputable def equiv
+local attribute [irreducible] power_basis.lift
+
+/-- `pb.equiv' pb' h₁ h₂` is an equivalence of algebras with the same power basis,
+where "the same" means that `pb` is a root of `pb'`s minimal polynomial and vice versa.
+
+See also `power_basis.equiv` which takes the hypothesis that the minimal polynomials are identical.
+-/
+noncomputable def equiv'
```

Comment:
Could we have a more descriptive name? Perhaps `equiv_of_aeval_minpoly`?

Reply:
`aeval_minpoly` is quite long, so I went for `equiv_of_root`. And renamed the other one to `equiv_of_minpoly`.